src/brevitas/core/scaling/runtime.py

```diff
@@ -168,21 +176,22 @@ class _AffineRescaling(brevitas.jit.ScriptModule):
     def __init__(
             self,
             scaling_shape,
-            shift_scale,
+            affine_weight_init: Optional[float],
+            affine_bias_init: Optional[float],
             dtype: Optional[torch.dtype],
             device: Optional[torch.device]):
         super(_AffineRescaling, self).__init__()
-        self.affine_weight = Parameter(torch.ones(scaling_shape, dtype=dtype, device=device))
-        if shift_scale:
+        self.affine_weight = Parameter(
+            affine_weight_init * torch.ones(scaling_shape, dtype=dtype, device=device))
```

Comment:
This init (and the next) seems a bit convoluted to me.

Since the typing of affine_weight_init is `Optional[float]`, we assume that the user will always pass a single scalar value.
Then we can initialize it as we do it for the parameter scaling:
```python
self.affine_weight = torch.full(scaling_shape, affine_weight_init, dtype=dtype, device=device)
```